src/cmd-visual/cmd-visuals.cpp

```diff
@@ -232,7 +232,7 @@ void do_cmd_visuals(PlayerType *player_ptr)
                 TERM_COLOR ca = r_ptr->x_attr;
                 byte cc = r_ptr->x_char;
 
-                term_putstr(5, 17, -1, TERM_WHITE, format(_("モンスター = %d, 名前 = %-40.40s", "Monster = %d, Name = %-40.40s"), r, r_ptr->name.data()));
+                term_putstr(5, 17, -1, TERM_WHITE, format(_("モンスター = %d, 名前 = %-40.40s", "Monster = %d, Name = %-40.40s"), static_cast<int>(r), r_ptr->name.data()));
```

Comment:
To convert an enum class to the corresponding integer type, use enum2i(). (and similarly in other places)